tests/test_lib.py

```diff
@@ -7,7 +7,7 @@
 from ops.charm import CharmBase
 from ops.framework import StoredState
 from ops.testing import Harness
-from lib.charms.prometheus.v1.prometheus import PrometheusConsumer
+from lib.charms.prometheus_k8s.v1.prometheus import PrometheusConsumer
```

Comment:
`lib` is in the python path and should be omitted here, i.e. should just be `charms....`
```
./run_tests:10:    export PYTHONPATH="lib:src"
```

Reply:
Ah thanks. I had a old version of `run_tests` and was missing this. Maybe charmcraft has been updated since then. will fix and push shortly.

Reply:
DONE.